modules/serverless-common/pages/service-mesh/common-service-mesh-network-isolation.adoc

```diff
@@ -0,0 +1,438 @@
+= Use {smproductname} to isolate network traffic with {serverlessproductname}
+:compat-mode!:
+// Metadata:
+:description: Use {smproductname} to isolate network traffic with {serverlessproductname}
+
+// TODO
+
+{smproductshortname} can be used to isolate network-traffic between "tenants" using {smproductshortname} `AuthorizationPolicy` resources. {serverlessproductname} can also leverage this, using several {smproductshortname} resources.
+
+
+.High-level architecture
+The high-level architecture consists of `AuthorizationPolicies` in the `knative-serving`, `knative-eventing` and the tenants namespaces, while all components are part of the {smproductshortname}. The respective `istio-proxy` sidecars take care of enforcing those rules to isolate traffic between tenants.
+
+*Knative Serving*
+
+image::service-mesh/multi-tenancy-service-mesh-Serving.drawio.svg[]
+
+*Knative Eventing*
+
+image::service-mesh/multi-tenancy-service-mesh-Eventing.drawio.svg[]
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+* You have set up {serverlessproductname} and {smproductname} as documented in xref:./common-service-mesh-setup.adoc[]
+
+
+.Securing the {smproductshortname}
+
+. As in the set-up documentation, make sure that all your tenants are part of the same `ServiceMeshMemberRoll` object as members:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+ name: default
+ namespace: istio-system
+spec:
+ members:
+   - knative-serving
+   - knative-eventing
+   - tenant-1
```

Comment:
> Also, its also technically not correct to call them "tenant-1" and "tenant-2" because both are members of the same tenant.

I think, whether they're par of the same tenant is define by AuthorizationPolicies later when the helm chart is generated. Not here in SMMR.

Reply:
Tip: I would probably avoid using "tenant-1" and "tenant-2" for names of projects unless we also want to use some prefix: "ns1-tenant1, "ns2-tenant1", "ns1-tenant2", "ns2-tenant2".

Reply:
Updated examples to be more clear.